src/tokens/transformation/web/formatCssAsJs.js

```diff
@@ -56,6 +56,7 @@ function formattedVariables (properties) {
 
 module.exports = ({ dictionary, file }) => {
   const themeObject = formattedVariables(dictionary.properties)
-  return fileHeader({ file }) +
-    'export default ' + JSON.stringify(themeObject, null, 2)
+  return fileHeader({ file }) + `
+import './variables.css'
```

Comment:
Hmmm, I don't know if we should do this here. I think consumers will need to import the CSS themselves, as they don't necessarily have support for importing `css` files and I don't think we want to force them to

Reply:
I guess that makes sense for vanilla non-bundled apps

Reply:
Agreed.

Reply:
I wonder if that module should therefore have a check to see if the variables have been added to the document. With the number of individual WebUI in brave-core and our intended other consumers, it could be easy to forget to include a request to that css file when adding a use of this theme, especially since it's not apparent it's using the css variables under the hood. It doesn't seem obvious for all devs to know "I'm using the css theme therefore I need to add a request to css variables." We could add a check something like:
```js
window.requestIdleCallback(() => {
  const value = getComputedStyle(document.documentElement).getPropertyValue('--spacing-4')
  if (!value) {
    console.warn(`Brave Leo css variables have not been added to this document and any use of Leo components or theme variables will be unstyled. Please add a request for '@brave/leo/css/variables.css'.`)
  }
}
```

Likewise, the same is true for the svelte components. Every time someone newly imports any component from Leo, they'll also need to add a request to the css variables. So instead of each svelte component directly importing, maybe they could directly import something like the above (named ~`cssVariableChecker.ts`)?

Reply:
I like this. Wonder if instead of something like `--spacing-4`, or anything else that could be found in other libraries, we should add a special var to our css output like `--leo-vars-present: true;`.

Reply:
Seems like a handy util to me :smile: Probably not critical enough to block this one :smile: